display/ssd1306.h

```diff
@@ -211,11 +216,20 @@ class SSD1306 : public I2CDevice, Looper, StateMachine, SaberBase, private Audio
 
       case SCREEN_MESSAGE:
         memset(frame_buffer_, 0, sizeof(frame_buffer_));
+    // Aurbesh Font option.
+    #ifdef AUREBESH
```

Comment:
we can avoid a little duplication here by doing something like:

#ifdef AURABESH
   const Glyph* font = Aurabsh10pt7Glyphs;
#else
   const Glyph* font = Starjedi10pt7bGlyphs;
#endif

Reply:
How about we use "USE_AUREBESH_FONT" instead of just "AURBESH" ?

Reply:
That is more descriptive, yes.

Reply:
I'm not sure how to use Glyph* font in place of the existing way it's coded, sorry. Is it used as an alias where the  DrawText line calls for the specified glyphs?

Reply:
You just use "font" instead of Aurabsh10pt7Glyphs/Starjedi10pt7bGlyphs

Reply:
done.

Reply:
Oh, this is generating this error when "font" is used in place of the the Glyphs :

In file included from /ProffieOS.ino:1627:
sketch/display/ssd1306.h: In member function 'int SSD1306::FillFrameBuffer()':
sketch/display/ssd1306.h:242:12: error: jump to case label
  242 |       case SCREEN_IMAGE:
      |            ^~~~~~~~~~~~
sketch/display/ssd1306.h:227:20: note:   crosses initialization of 'const Glyph* font'
  227 |       const Glyph* font = Starjedi10pt7bGlyphs;
      |                    ^~~~
exit status 1
Error compiling for board Proffieboard V2.

Reply:
Usually the way to fix this is to put a { after the case and a } somewhere before the "break".
This creates a scope for the "font" variable which doesn't cross any "case" jump targets.

Reply:
Fixed